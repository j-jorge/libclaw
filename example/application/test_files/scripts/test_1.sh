#!/bin/sh

./ex-application -b --string="my string" one two three
