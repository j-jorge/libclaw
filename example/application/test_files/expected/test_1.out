Testing error_level:
Testing warning_level:
Testing verbose_level:
'-b' argument is present.
'--string' argument value is 'my string'
